templates/cluster-template-topology.yaml

```
@@ -0,0 +1,55 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: '${CLUSTER_NAME}'
+  name: '${CLUSTER_NAME}'
+  namespace: '${NAMESPACE}'
+spec:
+  topology:
+    class: '${CLUSTER_CLASS_NAME}'
+    controlPlane:
+      replicas: ${CONTROL_PLANE_MACHINE_COUNT}
+    variables:
+    - name: sshKey
+      value: '${IONOSCLOUD_MACHINE_SSH_KEY}'
```

We already have the variable `IONOSCLOUD_MACHINE_SSH_KEYS`  and we would like to be able to provide multiple SSH keys

```suggestion
    - name: sshKeys
      value: '${IONOSCLOUD_MACHINE_SSH_KEYS}'
```